Libraries/src/Amazon.Lambda.Annotations/README.md

```diff
@@ -68,37 +187,141 @@ public class Functions
 }
 ```
 
-and place in your template:
+```json
+    "CloudCalculatorFunctionsAddGenerated": {
+      "Type": "AWS::Serverless::Function",
+      "Metadata": {
+        "Tool": "Amazon.Lambda.Annotations",
+        "SyncedEvents": [
+          "RootGet"
+        ]
+      },
+      "Properties": {
+        "Runtime": "dotnet6",
+
+...
+        "Role": {
+          "Fn::GetAtt": [
+            "LambdaRoleParameter",
+            "Arn"
+          ]
+        }
+      }
+    },
+```
+
+## Amazon API Gateway example
+
+This example creates a REST API through Amazon API Gateway that exposes the common arithmetic operations. 
+
+To avoid putting business logic inside the REST API a separate calculator service is created to encapsulated the logic of the arithmetic operations. Here is both the 
```

Comment:
To avoid putting business logic inside the REST API a separate calculator service is created to encapsulate the logic of the arithmetic operations. Here is both the 
calculator service's interface and default implementation. 
*Typo fix

Reply:
Fixed